x/ccv/provider/keeper/grpc_query_test.go

```diff
@@ -384,3 +402,105 @@ func TestQueryConsumerIdFromClientId(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, expectedConsumerId, res.ConsumerId)
 }
+
+// TestGetConsumerChain tests GetConsumerChain behaviour correctness
+func TestGetConsumerChains(t *testing.T) {
```

Comment:
Shall we rename to `TestQueryConsumerChains` and remove the comment because it seems redundant?